remembrance-day-piechart-bargraph/remembrance-day.ipynb

```diff
@@ -118,34 +123,36 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Bar Charts are used to diplay quantities among distint categoies. Unlike pie charts, however, the data can be overlapping. This bar chart shows both the quantity of those who enlisted and those who served oversease in WWI."
+    "Bar charts are used to display quantities among distinct categories. Unlike pie charts, however, the categories can have overlap. They don’t represent parts of a whole. In this case, every Canadian who served, enlisted. So the two categories could not be used to make a pie chart."
```

Comment:
I would edit as: 
"Bar Charts are used to compare quantities of different categories. This bar chart shows the quantity of Canadians who enlisted and and served overseas in WWI. Unlike pie charts, however, the categories can have overlap, so in this case, every Canadian who served also enlisted."